styleguide/_themes/derek/scss/components/subnav.scss

```diff
@@ -1,91 +1,158 @@
-//subnav
 .subnav {
-  background-color: $blue-gray;
+  background-color: $gray-dark;
   border-radius: 0;
+}
 
-  .nav > li > a:focus {
-    background-color: transparent;
-  }
+.subnav-container {
+  @include make-container;
+}
 
-  .nav > li.no-hover > a:focus {
-    background-color: $brand-success;
-  }
+.subnav-links {
+  list-style-type: none;
+  padding: 0;
+}
+
+.subnav-header {
+  float: left;
+  width: $full-width;
+}
+
+.subnav-collapseBtn {
+  @include make-xs-column(1);
+}
+
+.subnav-toggleBtn {
+  margin: 14px 0;
+  padding: 0;
 
-  .subnav-inside {
-    width: 100%;
+  .subnav-iconBar {
+    height: 3px;
   }
+}
 
-  .navbar-header {
+.subnav-pageTitleContainer {
+  @include make-xs-column(11);
+  display: none;
+  margin-top: 5px;
+}
 
-    .col-xs-10 {
-      margin-top: 10px;
-    }
+.subnav-pageTitle {
+  color: $white;
+}
 
-    button {
-      background-color: transparent;
-    }
+.icon-bar {
```

Comment:
@bighappyface The purpose of `bootstrap_overrides.scss` is to have a place for bootstrap fixes that can't be solved without excluding it from the sass linter (due to the need for weird nesting and element usage in the selector). We're trying to minimize the amount of code that gets excluded from sass lint, so we don't have a bunch of crappy code making it through, if possible.  Perhaps the file name is a bit misleading, and it should be called `no-sass-lint.scss` or something like that, but that is the point of it. 

That said, if we want somewhere to override bootstrap behavior, that is also run through the sass linter, then I'm totally fine with this idea. I would definitely agree that the `subnav.scss` file is not an appropriate place for this selector.